<ui version="4.0" >
 <class>UIMain</class>
 <widget class="QWidget" name="UIMain" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>405</width>
    <height>565</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>MVC</string>
  </property>
  <layout class="QGridLayout" >
   <property name="margin" >
    <number>5</number>
   </property>
   <property name="spacing" >
    <number>3</number>
   </property>
   <item row="1" column="0" colspan="2" >
    <widget class="QCheckBox" name="cbComplexModel" >
     <property name="text" >
      <string>Complex Model</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1" >
    <widget class="QPushButton" name="pbClose" >
     <property name="text" >
      <string>Close Project</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0" colspan="2" >
    <widget class="QTabWidget" name="tabWidget" >
     <property name="currentIndex" >
      <number>1</number>
     </property>
     <widget class="QWidget" name="pListView" >
      <attribute name="title" >
       <string>ListView</string>
      </attribute>
      <layout class="QVBoxLayout" >
       <property name="margin" >
        <number>5</number>
       </property>
       <property name="spacing" >
        <number>3</number>
       </property>
       <item>
        <widget class="QListView" name="listView" >
         <property name="editTriggers" >
          <set>QAbstractItemView::NoEditTriggers</set>
         </property>
         <property name="viewMode" >
          <enum>QListView::IconMode</enum>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="pTreeView" >
      <attribute name="title" >
       <string>TreeView</string>
      </attribute>
      <layout class="QVBoxLayout" >
       <property name="margin" >
        <number>5</number>
       </property>
       <property name="spacing" >
        <number>3</number>
       </property>
       <item>
        <widget class="QTreeView" name="treeView" >
         <property name="editTriggers" >
          <set>QAbstractItemView::NoEditTriggers</set>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="pTableView" >
      <attribute name="title" >
       <string>TableView</string>
      </attribute>
      <layout class="QVBoxLayout" >
       <property name="margin" >
        <number>5</number>
       </property>
       <property name="spacing" >
        <number>3</number>
       </property>
       <item>
        <widget class="QTableView" name="tableView" >
         <property name="editTriggers" >
          <set>QAbstractItemView::NoEditTriggers</set>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item row="0" column="0" >
    <widget class="QPushButton" name="pbOpen" >
     <property name="text" >
      <string>Open Project</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../resources/qmakeproject_resources.qrc" />
 </resources>
 <connections/>
</ui>
